src/v2/manifest/mod.rs

```diff
@@ -29,7 +32,7 @@ impl Client {
     ///
     /// The name and reference parameters identify the image.
     /// The reference may be either a tag or digest.
-    pub fn get_manifest_and_ref(&self, name: &str, reference: &str) -> FutureManifestAndRef {
+    pub fn get_manifest_and_ref(&self, name: String, reference: &str) -> FutureManifestAndRef {
```

Comment:
Can we keep the input type consistent between this and `get_manifest` (in whichever direction you prefer)?